.rstcheck.cfg

```diff
@@ -2,5 +2,5 @@
 ignore_directives = ifconfig
 # ignore the following languages because they require external compilers and
 # snippets in this repo depend on external code
-ignore_languages = c,cpp,c++
-ignore_substitutions = __PART_FAMILY_NAME__,__PART_FAMILY_DEVICE_NAMES__,__PRODUCT_LINE_NAME__,__SDK_BUILD_MACHINE__,__SDK_FULL_NAME__,__SDK_SHORT_NAME__,__SDK_INSTALL_FILE__,__SDK_INSTALL_DIR__,__SDK_DOWNLOAD_URL__,__LINUX_UBUNTU_VERSION_LONG__,__LINUX_UBUNTU_VERSION_SHORT__,__OPTEE_PLATFORM_FLAVOR__,__RTOS_UBUNTU_VERSION_LONG__,__WINDOWS_SUPPORTED_LONG__,__FEATURINGMATRIX__,__SYSFW_CORE_NAME__,__PCIE_BASE_ADDRESS__,__PCIE_DEVICE_ID__
+ignore_language = c,cpp,c++
```

Comment:
Key name regression. `ignore_languages` is the correct value.

Reply:
resolved